pypolychord/__init__.py

```diff
@@ -176,11 +176,17 @@ def run_polychord(loglikelihood, nDims, nDerived, settings,
     except ImportError:
         rank = 0
 
-    if not os.path.exists(settings.base_dir) and rank == 0:
-        os.makedirs(settings.base_dir)
-
-    if not os.path.exists(settings.cluster_dir) and rank == 0:
-        os.makedirs(settings.cluster_dir)
+    try:
+        if rank == 0:
+            os.makedirs(settings.base_dir)
+    except:
+        pass
+        
+    try:
+        if rank == 0:
+            os.makedirs(settings.cluster_dir)
+    except:
```

Comment:
Same here

Reply:
Done. Sorry, I forgot to change that the first time.